experimental/list_formatter/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "list_formatter"
+description = "ECMA-402 ListFormatter"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+# Keep this in sync with other crates unless there are exceptions
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+formatted_string_builder = { path = "../formatted_string_builder" }
+regex = "1.5"
+displaydoc = { version = "0.2.3", default-features = false }
+lazy_static = "1.0"
```

Comment:
Notes for later:

1. As discussed, `regex` should be replaced with `regex_automata`.
2. We should remove `lazy_static` once we adopt the data provider.